src/core/loading/Loader.tsx

```diff
@@ -14,17 +13,8 @@ interface LoaderProps {
 }
 
 export const Loader = (props: LoaderProps) => {
-  const overriddenDataModelUuid = useTaskStore((state) => state.overriddenDataModelUuid);
-  const overriddenTaskId = useTaskStore((state) => state.overriddenTaskId);
   const hasPresentation = useHasPresentation();
 
-  if (overriddenDataModelUuid) {
```

Comment:
All of these task-store checks in the loader was something we added to avoid the "page is loading" for each subform during the PDF render.

Have you tested how this now looks during PDF generation?

Reply:
Good point. Also counts for summaries of subforms.

Reply:
Nope, will check

Reply:
What do you think of this? As a compromise. Still showing the loader, but not the Presentation around it. 

Summary:
![image](https://github.com/user-attachments/assets/0cd302f1-2824-4606-8073-af8c1259d35f)

PDF:
![image](https://github.com/user-attachments/assets/652254a3-d20e-4074-beb9-19dd91783be2)

Reply:
Ready for print waits for subform data to be loaded. There should not be any difference really other than visual. It just shows a loader instead of nothing until the data is ready

Reply:
Unless i am missing something @danielskovli

Reply:
@bjosttveit @olemartinorg   Last time I was involved in this part of the code, the ready-for-print mechanism waited for all subform data to finish loading 👌 

The Loader modifications were only visual if I remember correctly. Worth noting that the `overriddenTaskId` check was already in there, I think perhaps @adamhaeger had added it as part of the Summary2 feature.

Reply:
`overriddenTaskId` is (or was?) needed to render summary of a previous task.

But there should be a cypress test for this and they all seem to be passing so IDK 😆

Reply:
I like the new skeleton loader 👍

Reply:
Still is, this will just show a loader instead of null while it loads. Due to `useHasPresentation` it should just show the loader and not the `Presentation` container around it. In PDF however we don't have any `Presentation` container, so I added a `DummyPresentation` that only sets the provider so that loading will not create its own `Presentation` there either